kvm/v12.2.0/README.md

```diff
@@ -0,0 +1,15 @@
+## :zap:  Giant Swarm Release 12.2.0 for KVM :zap:
+
+**If you are upgrading from 12.1.0, upgrading to this release will not roll your nodes.**
+
+As of this release **NGINX Ingress Controller App** is **optional and not pre-installed** component on KVM. This allows NGINX App installations to be managed independently from the base platform lifecycle. It is both benefit but also new responsibility to keep NGINX App installations up-to-date separately from rest of the cluster. Making NGINX optional enables use of other ingress controller alternatives without wasting resources where NGINX is not the preferred option. Upgrading existing tenant clusters with pre-installed NGINX will leave NGINX unchanged. Existing NGINX App custom resources will still have `giantswarm.io/managed-by: cluster-operator` label, but it will be ignored. The label will be cleaned up at a later point after all tenant clusters have been upgraded and KVM platform releases older than v12.2.0 archived.
```

Comment:
```suggestion
As of this release, **NGINX Ingress Controller App** is now an **optional (and not pre-installed)** component on KVM.

This enables use of alternative ingress controllers without wasting resources where NGINX is not the preferred option.

Now NGINX App installations can be managed and updated independent of the cluster, which is both a benefit and a new responsibility 😅

Upgrading tenant clusters with pre-installed NGINX will leave NGINX unchanged. Existing NGINX App custom resources will still have `giantswarm.io/managed-by: cluster-operator` label, but it will be ignored. The label will be cleaned up later after all tenant clusters have been upgraded and KVM platform releases older than v12.2.0 archived.
```